### DISCUSSION TITLE
PowerShell

### DISCUSSION THREAD

**Discussion by Jaisanj** [community/community#129237] [Programming Help]
### Body

There is a folder in each user's OneDrive named as K Drive,
I need a PowerShell script by which I can figure out the size of that folder in each user's OneDrive in a CSV File.

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by Tshiring** [comment]
You can use the Get-ChildItem cmdlet to list the items in the folder and then pipe the output to Measure-Object with the -Property Length -Sum parameter. Here’s an example:
Get-ChildItem -Path "C:\Users\<username>\OneDrive\K Drive" | Measure-Object -Property Length -Sum

 If you want to export the results to a CSV file, you can modify the command like this:
Get-ChildItem -Path "C:\Users\<username>\OneDrive\K Drive" | Measure-Object -Property Length -Sum | Export-Csv -Path "C:\path\to\output.csv" -NoTypeInformation

**Reply by Jaisanj** [reply]
Hello @Tshiring,
Thank you for your reply.
The cmdlet which you have shared is fetching the result from my File Explorer, If my OneDrive is synced. I am looking for the script by which I can fetch the size of the K drive folder for all the users in my tenant in a CSV file.

**Comment by ebndev** [comment]
Hi @Jaisanj, 

I'll be closing this discussion as there is a duplicate, newer [discussion](https://github.com/orgs/community/discussions/129308). In the future we ask that you don't double post, thanks!
